lib/Serge/Command/show.pm

```diff
@@ -0,0 +1,33 @@
+package Serge::Command::show;
+use parent Serge::Command;
+
+use strict;
+
+use Config::Neat::Inheritable;
+use Config::Neat::Render;
+
+sub get_commands {
+    return {
+        show => {
+            handler => \&run,
+            info => 'Show expanded version of a configuration file'
+        },
+    }
+}
+
+sub run {
+    my $config_file = $ARGV[0];
+
+    die "No configuration file specified\n" unless defined $config_file;
+    die "Multiple configuration files not allowed\n" unless $#ARGV == 0;
```

Comment:
This is not a proper way to test for multiple files, because one can provide a path to a directory in which .serge files are stored. The proper way is to run `get_config_files` method of `Serge::Application` and check that it's length is `1`. See this as an example: https://github.com/evernote/serge/blob/master/lib/Serge/Application.pm#L91-L94

This also requires the `need_config => 1` flag to be set in the command description (line 12 of this file).